File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::unordered_map<int, int> map; 
        for (auto& num: nums) {
            if (map.contains(num)) {
                return true;
            } else {
                map.insert({num, 0});
            }
        }

        return false;
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-1.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        len_nums = len(nums)
        ans = [0] * (2 * len_nums)

        for index, num in enumerate(nums): 
            ans[index] = num
            ans[index + len_nums] = num

        return ans